src/Cake.Core.Tests/Unit/TaskSetupContextTests.cs

```diff
@@ -13,8 +14,11 @@ public sealed class TheConstructor
             [Fact]
             public void Should_Throw_If_Task_Is_Null()
```

Comment:
Maybe we should add a test for checking that it throws if the context is null as well.